### PR TITLE
GEOPOR-852: petroleum SLD, vocab and parsing changes

### PR DESCRIPTION
**Pull request by GAAswinPeiris**

### CODE REVIEW

File: src/main/java/au/gov/geoscience/portal/config/vocabularies/VocabularyCacheList.java [hunk 1]
```diff
@@ -74,6 +74,18 @@ VocabularyServiceItem vocabularyTenementTypes(){
         return serviceItem;
     }
 
+    @Bean
+    VocabularyServiceItem vocabularyPetroleumTenementStatuses(){
+        VocabularyService service= new VocabularyService(serviceCaller, new VocabularyMethodMaker(), "http//resource.geoscience.gov.au/classifierScheme/ggic/2021.01/petroleum-tenement-status");
```
Comment (c-mckenna) [inline src/main/java/au/gov/geoscience/portal/config/vocabularies/VocabularyCacheList.java]: Should this be `http://`?

File: src/main/java/au/gov/geoscience/portal/config/vocabularies/VocabularyCacheList.java [hunk 2]
```diff
@@ -74,6 +74,18 @@ VocabularyServiceItem vocabularyTenementTypes(){
...
+        return new VocabularyServiceItem("vocabularyPetroleumTenementStatuses", "Petroleum Tenement Status Vocabulary", service);
+    }
+
+    @Bean
+    VocabularyServiceItem vocabularyPetroleumTenementTypes(){
+        VocabularyService service= new VocabularyService(serviceCaller, new VocabularyMethodMaker(), "http//resource.geoscience.gov.au/classifierScheme/ggic/2021.01/petroleum-tenement-type");
```
Comment (c-mckenna) [inline src/main/java/au/gov/geoscience/portal/config/vocabularies/VocabularyCacheList.java]: Same here?

File: src/main/java/au/gov/geoscience/portal/config/vocabularies/VocabularyCacheList.java (hunk 1 above)
Comment (GAAswinPeiris) [reply src/main/java/au/gov/geoscience/portal/config/vocabularies/VocabularyCacheList.java]: Fixed

File: src/main/java/au/gov/geoscience/portal/config/vocabularies/VocabularyCacheList.java (hunk 2 above)
Comment (GAAswinPeiris) [reply src/main/java/au/gov/geoscience/portal/config/vocabularies/VocabularyCacheList.java]: Fixed

**Review by c-mckenna — APPROVED** [review]

**Pull request merged**